util/feature.txt

```diff
@@ -0,0 +1,3 @@
+feature1
+feature2
```

Comment:
let's add a bit more to the feature2 implementation . I think it should handle multiple weights